src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.cs

```diff
@@ -107,7 +107,7 @@ public void TestCreateAndGetSoftwareUpdateConfigurationWithPrePost()
             TestRunner.RunTestScript("Test-CreateAndGetSoftwareUpdateConfigurationWithPrePost");
         }
 
-        [Fact]
+        [Fact(Skip = "Temporarily skipping")]
```

Comment:
I think this test shouldn't skip here, should it?

Reply:
Did it temporarily, have updated the recording for this test.